static/app/stores/Commiters/CommittersContext.tsx

```diff
@@ -0,0 +1,279 @@
+import {createContext, useContext, useEffect, useReducer} from 'react';
+
+import {Client} from 'sentry/api';
+import {
+  AvatarProject,
+  Committer,
+  Event,
+  Group,
+  Organization,
+  Project,
+} from 'sentry/types';
+import getDisplayName from 'sentry/utils/getDisplayName';
+import useApi from 'sentry/utils/useApi';
+
+function exhaustive(x?: never) {
+  throw new Error(
+    `Unhandled ${JSON.stringify(x)} switch case action in committersReducer`
+  );
+}
+
+function makeCommitterStoreKey({
+  organizationSlug,
+  projectSlug,
+  eventId,
+}: {
+  eventId: string;
+  organizationSlug: string;
+  projectSlug: string;
+}): string {
+  return `${organizationSlug} ${projectSlug} ${eventId}`;
+}
+
+export function fetchCommitters(
```

Comment:
```suggestion
 function fetchCommitters(
```

it seems that this is only being used in this file